### ISSUE TITLE
[ERRMSG] Enhance eosio-launcher Error-Messages

### ISSUE THREAD

**Issue by lazaridiscom** [EOSIO/eos#4904]
~[I'm trying to provide a patch for this during this night.]~ the path-related problem has disappeared (using ubuntu 18.04 and latest master). Still, the error message could be more precise.

Several users have reported a failing build-validation.

In at least 2 instances (https://github.com/EOSIO/eos/issues/4765#issuecomment-408467908), the eosio-launcher reports `is not a valid path`.

This error-message should be enhanced, ideally to the point that it describes the root-cause.

```
Test project /home/dex/github/eos/build
    Start 31: nodeos_sanity_test
1/1 Test #31: nodeos_sanity_test ...............***Failed    0.18 sec
 is not a valid path
 is not a valid path
 is not a valid path
 BEGIN
 UTC time: 2018-07-27 16:16:54.727825
 EOS Client version: 40a20769
 Processor: x86_64
 OS name: Linux-4.15.0-23-generic-x86_64-with-Ubuntu-18.04-bionic
SERVER: localhost
PORT: 8888
 cmd: pkill -9 keosd
 cmd: programs/eosio-launcher/eosio-launcher -k 9
 cmd: pkill -9 nodeos
Stand up cluster
  Checking if port 8888 is available.
  Checking if port 8889 is available.
 cmd: programs/eosio-launcher/eosio-launcher -p 1 -n 1 -s mesh -d 1 -i 2018-07-27T16:16:54.752 -f --p2p-plugin net --nodeos --max-transaction-time 5000 --abi-serializer-max-time-ms 5000 --filter-on * --p2p-max-nodes-per-host 1 --contracts-console
 ERROR: Launcher failed to launch.
 FAILURE - launcher
 ERROR: Failed to stand up eos cluster.
 Test failed.
 cmd: pkill -9 keosd
 cmd: programs/eosio-launcher/eosio-launcher -k 9
 cmd: pkill -9 nodeos
```

**Comment by taokayan** [comment]
please set $EOSIO_HOME to your build folder

**Comment by lazaridiscom** [comment]
fixed with #5525
